go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/elastic/package-registry
 
-go 1.22.9
+go 1.23.0
```

Comment:
It was required to update this go version. Buildkite failure link:
https://buildkite.com/elastic/package-registry/builds/801#0194f868-eddb-4be1-9a77-39d4b5405af2

FYI @jsoriano